sdk/nodejs/api/utils.ts

```diff
@@ -35,7 +35,7 @@ export function queryBuilder(q: QueryTree[]) {
   })
   query += "}"
 
-  return query.replace(/\s+/g, "")
+  return query;
```

Comment:
```suggestion
  return query.trim();
```

We probably still want whitespace trimmed at the start and end of args, maybe?